fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -266,6 +267,11 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     public static final String ENABLE_QUERY_DEBUG_TRACE = "enable_query_debug_trace";
 
     public static final String PARSE_TOKENS_LIMIT = "parse_tokens_limit";
+    public static final String ENABLE_QUERY_CACHE = "enable_query_cache";
+    public static final String QUERY_CACHE_FORCE_POPULATE = "query_cache_force_populate";
+    public static final String QUERY_CACHE_ENTRY_MAX_BYTES = "query_cache_entry_max_bytes";
+    public static final String QUERY_CACHE_ENTRY_MAX_ROWS = "query_cache_entry_max_rows";
+    public static final String ENABLE_TEST_MODE = "test_mode";
```

Comment:
remove it

Reply:
commented

Reply:
you could use FeConstants.runningUnitTest